src/backend/asset-status/index.ts

```diff
@@ -127,5 +126,21 @@ export function updateAssetStatusSS({
         }
     }
 
-    messaging.subscribe(TOPIC, WaitLoop);
+    function subscribeAndInitialize(topics: string[]): void {
```

Comment:
Can we move this function to the Normalizer and change the function signature to accept a reference to the `WaitLoop` ? That way we're not duplicating this logic in each module

Reply:
Done